ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-reviewer.yml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-ci-reviewer
+spec:
+  params:
+    - name: pipeline_image
+    - name: git_username
+    - name: operator_name
+    - name: pr_url
+    - name: github_token_secret_key
+      description: The key within the Kubernetes Secret that contains the GitHub token.
+      default: token
+    - name: github_token_secret_name
+      description: The name of the Kubernetes Secret that contains the GitHub token.
+      default: github
+  workspaces:
+    - name: repo
+      description: Cloned repository
+  results:
+    - name: author_is_reviewer
+      description: >-
+        Check if the author of the PR is listed also as a reviewer
+    - name: list_reviewers
+      description: >-
+        Return list of all reviewers from ci.yaml file except author of the pull request
+  steps:
+    - name: get-ci-reviewer
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.repo.path)
+      script: |
+        #! /usr/bin/env bash
+        set -e
+        # do not use -x to not leak github secrets into logs!!!
+        get-ci-reviewer \
+          --repo-path="$(workspaces.repo.path)" \
+          --git-username="$(params.git_username)" \
+          --operator-name="$(params.operator_name)" \
+          --output-file /tmp/output.json \
+          --verbose
+        list_of_reviewers=$(jq -r '.all_reviewers[]' /tmp/output.json)
+        bool_reviewer="$(jq -r '.is_reviewer' /tmp/output.json)"
+
+        # redirecting results variables to /dev/null so they would not be printed
+        echo -n "$list_of_reviewers" | tee $(results.list_reviewers.path) > /dev/null
+        echo -n "$bool_reviewer" | tee $(results.author_is_reviewer.path) > /dev/null
```

Comment:
We have the `yq` tool in the container image: we don't need the python script at all to do this:

```sh
CI="operators/$(params.operator_name)/ci.yaml"
if [ -e $CI ] ; then
  yq -r '.reviewers' $CI >$(results.list_reviewers.path)
  yq -r ".reviewers|index(\"$(params.git_username)\") != null" $CI >$(results.author_is_reviewer.path)
else
  echo >$(results.list_reviewers.path)
  echo false >$(results.author_is_reviewer.path)
fi
```